packages/core/src/components/dialog/_dialog.scss

```diff
@@ -15,7 +16,7 @@ Markup:
   <div class="#{$ns}-dialog">
     <div class="#{$ns}-dialog-header">
       <span class="#{$ns}-icon-large #{$ns}-icon-inbox"></span>
-      <h4 class="#{$ns}-heading">Dialog header</h4>
+      <h5 class="#{$ns}-heading">Dialog header</h5>
```

Comment:
This is a minor change to the generated DOM, but it should generally be backwards compatible (won't break existing layouts), and anyone using the React component will get the new styles.